test/cloudtest/pkg/providers/packet/packet_provider.go

```diff
@@ -656,19 +656,9 @@ func (p *packetProvider) ValidateConfig(config *config.ClusterProviderConfig) er
 	for _, envVar := range config.EnvCheck {
 		envValue := os.Getenv(envVar)
 		if envValue == "" {
-			return errors.Errorf("environment variable are not specified %s Required variables: %v", envValue, config.EnvCheck)
+			return errors.Errorf("environment variable not set: %s. Required variables: %v", envVar, config.EnvCheck)
 		}
 	}
 
-	envValue := os.Getenv("PACKET_AUTH_TOKEN")
```

Comment:
We need to have this options to be checked, since user could miss to specify them in config.

Reply:
[x] hard-coded check for "PACKET_AUTH_TOKEN", "PACKET_PROJECT_ID"